ui/src/views/Rules.vue

```diff
@@ -219,6 +220,52 @@
               </div>
             </div>
           </div>
+          <div v-if="r.type === 'separator'" v-show="searchString.length == 0" class="list-view-pf-checkbox drag-here">
+            <span class="fa fa-bars"></span>
+          </div>
+          <div v-if="r.type === 'separator'" class="list-view-pf-actions">
+            <div class="dropup pull-right dropdown-kebab-pf">
+              <button
+                class="btn btn-link dropdown-toggle"
+                type="button"
+                id="dropdownKebabRight9"
+                data-toggle="dropdown"
+                aria-haspopup="true"
+                aria-expanded="true"
+              >
+                <span class="fa fa-ellipsis-v"></span>
+              </button>
+              <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownKebabRight9">
+                <li @click="openEditSeparator(r)">
+                  <a>
+                    <span class="fa fa-edit span-right-margin"></span>
+                    {{$t('edit')}}
+                  </a>
+                </li>
+                <li @click="openDeleteSeparator(r)">
+                  <a>
+                    <span class="fa fa-times span-right-margin"></span>
+                    {{$t('delete')}}
+                  </a>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div v-if="r.type === 'separator'" class="list-view-pf-main-info small-list">
+            <div class="list-view-pf-left">
+            </div>
+            <div class="list-view-pf-body">
+              <div class="list-view-pf-description rules-src-dst">
+                <div class="list-group-item-heading">
+                <span>{{r.Description.toUpperCase()}}</span>
+                </div>
+                <div class="list-group-item-text">
+                </div>
+              </div>
+              <div class="list-view-pf-additional-info rules-info">
+              </div>
+            </div>
+          </div>
```

Comment:
```suggestion
            </div>
          </template>
```
Just closing the `template` element, see comment above